infra/build/functions/build_and_run_coverage.py

```diff
@@ -173,6 +174,19 @@ def get_build_steps(  # pylint: disable=too-many-locals, too-many-arguments
       ],
   })
 
+  build_steps.append(build_lib.gsutil_rm_rf_step(upload_report_by_target_url))
+  build_steps.append({
+      'name':
+          'gcr.io/cloud-builders/gsutil',
+      'args': [
+          '-m',
+          'cp',
+          '-r',
+          os.path.join(build.out, 'report_target'),
```

Comment:
nit: `target_report` sounds better to me.

Reply:
This isn't really visible to users, and is a bit annoying to change at this point (requires a base image change first). Can change later.